orchestra/static/orchestra/todos/todo-list.directive.es6.js

```diff
@@ -27,19 +27,22 @@ export default function todoList (orchestraApi) {
       todoList.templates = []
       todoList.todoQas = []
 
-      const createTodo = (taskId, title, completed, startDate, dueDate) => todoApi.create({
-        task: taskId,
-        title,
-        completed,
-        start_by_datetime: startDate,
-        due_datetime: dueDate
-      }).then((taskData) => {
-        todoList.todos.unshift(taskData)
-        return taskData
-      })
+      const createTodo = (title, completed, startDate, dueDate) => {
```

Comment:
why did taskId get passed in, but now we'd get the step slug from `todoList.newTodoStepSlug`?

Reply:
(not saying it's wrong, just making sure i understand)

Reply:
```
13         <select class="form-control input-sm" name="todoList" id="todoList" ng-model="todoList.newTodoStepSlug">
 14           <option value="" selected>Select owner</option>
 15           <option value="{{task.step_slug}}" ng-repeat="task in todoList.possibleTasks">{{todoList.steps[task.step_slug].name}}</option>
 16         </select>
```
We set a selected `step_slug` to `newTodoStepSlug` here (in `orchestra/static/orchestra/todos/todo-list.html`).
We used to require a task.id to create a todo, but don't need it anymore.

Reply:
Hahha, I get your question now. Sorry.

I grepped for `sendToPending`. It is not being used anywhere 🙈

Reply:
It was deleted in this PR (2 years ago): https://github.com/b12io/orchestra/pull/451/files#diff-0decc76f5bde80e17d109210f95a37ae